lib/redis-session-store.rb

```diff
@@ -128,6 +128,11 @@ def set_session(env, sid, session_data, options = nil)
   end
   alias write_session set_session
 
+  def get_expiry(env, options)
+    session_storage_options = options || env.fetch(ENV_SESSION_OPTIONS_KEY)
```

Comment:
The existing code seems to assume that `env.fetch(ENV_SESSION_OPTIONS_KEY)` always exists, since I just cut/pasted that line. Happy to be a little defensive though.